### DISCUSSION TITLE
Account Manager Plugin

### DISCUSSION THREAD

**Discussion by skifree0** [runelite/runelite#15134] [Ideas]
A version or a rip of the account manager that existed on 3rd party clients. You could save full account details, so email and pass. Please and thank you.

**Comment by Nightfirecat** [comment]
Per #10145, we are not interested in storing passwords to make this possible. In the short term, we recommend using a password manager which can auto-type your account credentials. In the long term, the Jagex launcher (and RuneLite along with it) will support having multiple characters available to log in from a single account.
